b2luigi/batch/processes/htcondor.py

```diff
@@ -44,8 +45,16 @@ def _ask_for_job_status(self, job_id: int = None):
         q_cmd = ["condor_q", "-json", "-attributes", "ClusterId,JobStatus,ExitStatus"]
 
         if job_id:
-            output = subprocess.check_output(q_cmd + [str(job_id)])
-        else:
+            q_cmd = q_cmd + [str(job_id)]
+
+        try:
+            output = subprocess.check_output(q_cmd)
+        except:
```

Comment:
your `except` catches all exceptions, but I would be more specific. Which is the exception that we're actually getting? This would be also useful to extend the `@retry` decorator above